test/e2e_node/cpu_manager_test.go

```diff
@@ -2798,9 +3532,12 @@ func makeCPUManagerInitContainersPod(podName string, ctnAttributes []ctnAttribut
 
 type cpuManagerKubeletArguments struct {
 	policyName                       string
+	topologyManagerPolicy            string
+	topologyManagerScope             string
```

Comment:
this is a pretty strong smeel as cpumanager behavior should NOT depend on topology manager, because the topology manager sits higher in the stack.